modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -5,6 +5,12 @@ class Record < ::Proxy::Dns::Record
     include Proxy::Log
     include Proxy::Util
 
+    private
+    def popen3(*args)
+      Open3.popen3(*args)
+    end
+
+    public
```

Comment:
It is a common pattern to start with public methods first and move private section towards the end of the class, so you do not need to use `public` keyword at all.